yarn-project/blob-sink/src/server/server.ts

```diff
@@ -69,6 +71,12 @@ export class BlobSinkServer {
   private async handleGetBlobs(req: Request, res: Response) {
     const { blobHashes: blobHashesQuery } = req.query;
 
+    if (this.disableStorage) {
+      this.log.warn(`Blob storage is disabled, cannot retrieve blobs`);
+      res.json({ version: 'deneb', data: [] });
+      return;
```

Comment:
The only thing I'd say is this ends up being an error response in Lighthouse rather than an empty response